examples/parse_time.py

```diff
@@ -0,0 +1,94 @@
+"""
+================================================
+Parsing times with sunpy.time.parse_time
+================================================
+
+Example to show some example usage of parse_time
+"""
+##############################################################################
+# Import the required modules
+from datetime import datetime, date
+import time
+
+import numpy as np
+import pandas
+
+# Import parse_time
+from sunpy.time import parse_time
+
+
+##############################################################################
+# Suppose you want to parse some strings, ``parse_time`` can do that.
+t1 = parse_time('1995-12-31 23:59:60')
+
+##############################################################################
+# Of course you could do the same with astropy ``Time``.
```

Comment:
Should intersphinx to Astropy Time here.

Reply:
Inter sphinx works as `~astropy.time.Time`, right?